trade-finance-manager-ui/templates/case/facility/facility.njk

```diff
@@ -8,7 +8,7 @@
 
 {% block pageTitle -%}
   Deal {{ deal.details.ukefDealId }} Facility {{ facility.ukefFacilityId }}
-{%- endblock %}
+{%- endblock pageTitle %}
```

Comment:
I'd rather expressively state what block is ending -- It's more adaptable.

Reply:
Not a fan of this, this would be different to everywhere else, also it's only a block with 1x line, I think it's pretty clear

Reply:
reverted :)